### ISSUE TITLE
ChoiceType generates options in selectbox with parent's attributes

### ISSUE THREAD

**Issue by ossinkine** [symfony/symfony#16534]
I've created `TestFormType`

``` php
class TestType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder->add('choice', 'choice', ['choices' => ['foo', 'bar']]);
    }

    public function getName()
    {
        return 'test';
    }
}
```

created the form in the controller and passed to the view

``` php
class DefaultController extends Controller
{
    public function indexAction(Request $request)
    {
        return $this->render('AppBundle::index.html.php', [
            'form' => $this->createForm('test')->createView(),
        ]);
    }
}
```

rendered in the view

``` php
<?php echo $view['form']->widget($form['choice']) ?>
```

and I got this:

``` html
<select id="test_choice" name="test[choice]">
    <option value="0" id="test_choice" name="test[choice]" required="required">foo</option>
    <option value="1" id="test_choice" name="test[choice]" required="required">bar</option>
</select>
```

Options have the same attributes as select. Options shoud not have them.

I'm working on the latest version of 2.7 branch

**Comment by xabbuh** [comment]
@ossinkine Can you provide a fork of the Symfony Standard Edition that reproduces your issue?

**Comment by ossinkine** [comment]
@xabbuh https://github.com/ossinkine/symfony-standard/tree/issue-16534

**Comment by ossinkine** [comment]
@webmozart You have added rendering parent's attributes (https://github.com/symfony/symfony/commit/03efce1b568379eac21d880e427090e43035f505#diff-b56b980552f37a32080a57032805f34dR11). Could you please explain for what did you do this and how should it work?

**Comment by xabbuh** [comment]
This seems to have been fixed in the meantime. I could reproduce your issue with the Symfony version locked in your example project (i.e. 2.7.6), but it was resolved updating to Symfony 2.7.29 (which is the latest patch release for Symfony 2.7).

**Comment by xabbuh** [comment]
probably solved by #19020 which fixed #19016

**Comment by ossinkine** [comment]
I confirm, fixed in 2.7.19
